xcube/core/store/descriptor.py

```diff
@@ -154,7 +154,7 @@ def from_dict(cls, d: Mapping[str, Any]) -> 'DatasetDescriptor':
                                  data_vars=data_vars,
                                  attrs=d.get('attrs', None),
                                  open_params_schema=d.get('open_params_schema', None),
-        )
+                                 )
```

Comment:
Cannot fix. In the version that I have checked out, the bracket is placed correctly.

Reply:
Sorry, but do you see the ending bracket in the wrong position after you have checked this part out? Because I just realize the diff view above states that I FIXED the layout by putting the bracket in the correct place.

Reply:
Let's ignore this....